src/scss/components/_widget.scss

```diff
@@ -18,6 +18,13 @@
   }
 
   &__title {
+    @include media(small) {
+      font-size: 24px;
+    }
+
+    font-size: 20px;
+    font-style: italic;
+    font-weight: 900;
```

Comment:
@eduardojmatos Do we have a font-weight map?

Reply:
@iondrimba Yes. There is a get-weight, I didn't know about this function haha